cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -272,6 +272,9 @@
 
     public static final String SERVICE_DESCRIPTION = "Service that handles stream data ingestion.";
     /* End constants used by stream */
+
+    // Delay in seconds between two heartbeats in a stream service
+    public static final int HEARTBEAT_DELAY = 2;
```

Comment:
Should it be delay or should it be interval? The difference is that delay is the time between last run completion to next run start, while interval is the time between start time of two runs.